packages/client-js/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "rimraf build && yarn build:manifest && tsc",
     "build:manifest": "node ./scripts/generateManifest.js",
+    "lint:code": "eslint --color --ext .ts src/",
```

Comment:
This will by default use the .eslintrc.js file at the root folder? cool!

Reply:
It will use nearest .eslintrc.js and combine with every .eslintrc.js until it finds one with `{root: true}`